modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/AnalyticsCloudModal/index.js

```diff
@@ -39,16 +41,33 @@ const AnalyticsCloudModal = ({
 					leftButton="Cancel"
 					onClose={onClose}
 					project={project}
+					setFormAlreadySubmitted={setFormAlreadySubmitted}
 					subscriptionGroupId={subscriptionGroupId}
 				/>
 			),
 		}),
-		[onClose, project, subscriptionGroupId]
+		[onClose, project, setFormAlreadySubmitted, subscriptionGroupId]
 	);
 
+	const submittedModalTexts = {
+		paragraph:
+			'Return to the product activation page to view the current Activation Status',
+		subtitle: `We'll need a few details to finish building your Analytics Cloud workspace(s).`,
+		text:
+			'Another user already submitted the Analytics Cloud activation request.',
+		title: 'Set up Analytics Cloud',
+	};
+
```

Comment:
Hi @igordmf, constant values should be good created before of component as not depend the component render

Reply:
Done! Thank you!